ci-operator/step-registry/openshift/e2e/vsphere/csi/migration/openshift-e2e-vsphere-csi-migration-workflow.yaml

```diff
@@ -20,5 +20,13 @@ workflow:
         Managed cluster should start all core operators\|
         Alerts shouldn't report any alerts in firing or pending state\|
         Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (default fs)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (default fs)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (ext4)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (default fs)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (ext4)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (ext4)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (ext4)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (default fs)] volumes should store data
```

Comment:
Could this be shortened to some like this?
```
In-tree Volumes \[Driver: vsphere\] \[Testpattern: Inline-volume
In-tree Volumes \[Driver: vsphere\] \[Testpattern: Pre-provisioned PV
```

Reply:
updated the regular expression after I realized, it has to be a single one. Not multiple lines.